pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -176,7 +178,9 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 				IdleTimeout: fi.Int64(int64(idleTimeout.Seconds())),
 			},
 
-			Tags: tags,
+			Tags:             tags,
+			NLBNamesToDelete: []string{b.NLBName("api")},
+			TGNamesToDelete:  []string{b.NLBTargetGroupName("tcp"), b.NLBTargetGroupName("tls")},
```

Comment:
I would prefer that these are not passed like this and just hardcoded in `FindDeletions`.

Reply:
@rifelpet any thoughts?

Reply:
we explored this, but it would require either refactoring these methods and the methods they depend on to not depend on the model and move them to a common utility package that could be imported by both awsmodel and awstasks, or duplicate the logic which is pretty significant (NLBTargetGroupName calls [this](https://github.com/kubernetes/kops/blob/e91ed114499294f3e4242c6b23e7150a5c7262ec/pkg/model/context.go#L57-L99) for example)

Given that these fields wont ever be shown in the dryrun output I don't think its that bad of a solution, but maybe the refactoring would be a better idea.

Reply:
I think would be best either to refactor or to just make a copy of `GetELBName32()` in https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/awsup/aws_utils.go.

Also, not sure if we should detect the ELB by name or by tags, but name should be ok for now.